vmdb/app/models/ems_microsoft.rb

```diff
@@ -44,10 +44,16 @@ def verify_credentials(_auth_type = nil, options = {})
     raise MiqException::MiqHostError, "No credentials defined" if self.authentication_invalid?(options[:auth_type])
 
     begin
-      run_dos_command("hostname")
+      run_dos_command("powershell import-module virtualmachinemanager; " \
+      "Get-SCVMMServer -ComputerName localhost")
+
+    rescue RuntimeError => e
+      raise MiqException::MiqHostError, "#{e.message}"
```

Comment:
I find it weird to be rescuing a RuntimeError...I feel like the other method should raise something specific